cumulusci/tasks/salesforce/EnsureRecordTypes.py

```diff
@@ -74,6 +79,11 @@ def _init_options(self, kwargs):
                 "Record Type Developer Name value must contain only alphanumeric or underscore characters"
             )
 
+        # Validate Description has 255 characters
+        self.options["record_type_description"] = (
+            str(self.options.get("record_type_description") or "")
+        )[:255]
```

Comment:
It might be better to raise a TaskOptionsError if the value is too long, but it's probably fine either way.

Reply:
I went the simpler-to-test route.   Should I change this?

Reply:
Nah, it's fine. I assume this is more to provide some context and it's not critical to reproduce a long description faithfully.